source/js/components/search-form.js

```diff
@@ -0,0 +1,101 @@
+import fetchUtils from "@degjs/fetch-utils";
+
+const searchForm = () => {
+
+
+    const formEl = document.querySelector('.js-search-form');
+    const layoutEl = document.querySelector('.js-general-content')
+
+    const url = formEl.dataset.api;
+
+    function init() {
+        formEl.addEventListener('submit', e => {
+            e.preventDefault();
+            getData();
+        });
+    }
+
+
+    function getData() {
+        const formEls = formEl.querySelectorAll('.js-value');
```

Comment:
[fix] Use ES6 spread to convert a NodeList to an actual array

Reply:
I had to remove SelectorAll since there is a single input. I will make sure to use spread when selecting multiple elements. Thanks.